pom.xml

```diff
@@ -306,25 +306,28 @@
 							</execution>
 						</executions>
 					</plugin>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-javadoc-plugin</artifactId>
-						<version>3.0.1</version>
-						<executions>
-							<execution>
-								<id>attach-javadocs</id>
-								<phase>verify</phase>
-								<goals>
-									<goal>jar</goal>
-								</goals>
-							</execution>
-						</executions>
-						<configuration>
-							<quiet>true</quiet>
-							<doclint>none</doclint> 
-							<additionalparam>-Xdoclint:none</additionalparam>
-						</configuration>
-					</plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.11.1</version>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <quiet>true</quiet>
+                            <doclint>none</doclint>
+                            <failOnWarnings>true</failOnWarnings>

```

Comment:
Just another rabbit nosense.

With `doclint:none` there are only some minimal checks:

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:69: warning: Tag @link: reference not found: #Tabs(Tab...)
[WARNING] * the {@link #Tabs(Tab...)} constructor. The Tab components added to it can be selected with the
[WARNING] ^
[WARNING] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:72: warning: Tag @link: reference not found: #Tabs(boolean, Tab...)
[WARNING] * autoselection is explicitly disabled with {@link #Tabs(boolean, Tab...)}, or {@link
[WARNING] ^
[WARNING] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:164: warning: Tag @link: reference not found: #Tabs(boolean, Tab...)
[WARNING] * is explicitly disabled with {@link #Tabs(boolean, Tab...)}, or {@link #setAutoselect(boolean)}.
[WARNING] ^
[WARNING] 3 warnings
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.11.1:jar (attach-javadocs) on project enhanced-tabs-addon: MavenReportException: Error while generating Javadoc: Project contains Javadoc Warnings
```

OTOH without default `doclint` settings

```
[ERROR] Exit code: 1
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:69: error: reference not found
[ERROR]  * the {@link #Tabs(Tab...)} constructor. The Tab components added to it can be selected with the
[ERROR]               ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:72: error: reference not found
[ERROR]  * autoselection is explicitly disabled with {@link #Tabs(boolean, Tab...)}, or {@link
[ERROR]                                                     ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:164: error: reference not found
[ERROR]    * is explicitly disabled with {@link #Tabs(boolean, Tab...)}, or {@link #setAutoselect(boolean)}.
[ERROR]                                         ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:222: warning: no comment
[ERROR]   public RouterLink addRouterLink(String text, Class<? extends Component> target) {
[ERROR]                     ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:589: warning: no comment
[ERROR]   public int getTabCount() {
[ERROR]              ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:593: warning: no comment
[ERROR]   public Stream<Tab> getTabs() {
[ERROR]                      ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:457: warning: no comment
[ERROR]   public void setSelectedTab(Tab selectedTab, boolean changedFromClient) {
[ERROR]               ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:90: warning: no comment
[ERROR]   private boolean autoselect = true;
[ERROR]                   ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:92: warning: no comment
[ERROR]   private int selectedIndex = -1;
[ERROR]               ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:354: warning: no comment
[ERROR]     private final boolean initialSelection;
[ERROR]                           ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:353: warning: no comment
[ERROR]     private final Tab previousTab;
[ERROR]                       ^
[ERROR] J:\fc\addons\EnhancedTabs\src\main\java\com\flowingcode\vaadin\addons\enhancedtabs\EnhancedTabs.java:352: warning: no comment
[ERROR]     private final Tab selectedTab;
```